superset-frontend/src/components/Select/utils.tsx

```diff
@@ -25,6 +25,12 @@ import { LabeledValue, RawValue, SelectOptionsType, V } from './types';
 
 const { Option } = AntdSelect;
 
+export const SELECT_ALL_VALUE = 'Select All';
```

Comment:
For compatibility reasons, I think in this first phase we should send all values to `onChange` when selecting "Select all". It would be the same as our current behavior where a user can manually select all items. In the next phase, we should come up with a smarter way of representing the "Select All" value but this will require backend changes to interpret this value. My [original PR](https://github.com/apache/superset/pull/20143) has an item about this:

> Handle the payload size. We shouldn't send all selected items to the server-side because we don't know how many can exist, which could result in huge payload size. What we need to do is to come up with a payload that represents the user selection in an efficient way. One example would be if a user selects all items and unselects a subset, we could send a value representing the "Select all" alongside the subset of unselected items.